airbyte-integrations/connectors/source-dremio/source_dremio/schemas/catalogs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "entityType": {
+      "type": "string"
+    },
+    "id": {
+      "type": "string"
+    },
+    "type":{
+      "type": "string"
+    },
+    "path": {
+      "type": "array"
+    },
+    "createdAt":{
+      "type": "string"
+    },
+    "tag": {
+      "type": "string"
+    },
+    "sql": {
```

Comment:
the schema does not match the API response

Reply:
changed this as well